app/javascript/src/components/editor/CodeMirror.svelte

```diff
@@ -69,48 +117,32 @@
 
     requestAnimationFrame(() => {
       updatingState = false
-      setScrollPosition()
+      setScrollPosition(view, currentId)
     })
   }
```

Comment:
Was this really the best way to do this? Why do we need to overwrite the scroll position each frame, and not only when the editor is being initialized/the user goes to another file?

Reply:
True, it might be a bit much. Since it's not related to this PR I will ignore for now, but I will create an issue so we don't forget